calebwagner/kneel-diamonds
# fixed bug on main module

I fixed the big on the main module. "renderAllHTML" was never invoked in the addEventListener function.

Steps to check code:
1. git fetch --all
2. git checkout cjw-breaking-code
3. code .
4. check the main module to make sure "renderAllHTML" is invoked
5. serve
6. check to make sure the event listener is working correctly by placing a complete order without an alert pop up window appearing
7. check to see if the new order appeared at the bottom of the page
